README.md

```diff
@@ -40,4 +40,8 @@ from the README.md leaving only your finished README content for your project. -
 
 # Project Name
 
-<!-- TODO: Edit the project name heading and include a description of the project -->
\ No newline at end of file
+<!-- TODO: Edit the project name heading and include a description of the project -->
+
+# License
+$PROJECT_NAME is licensed under an [Apache 2.0 license](./LICENSE).
+The #PROJECT_NAME documentation is licensed under a [CC-BY-4.0 license](./LICENSE-docs). 
```

Comment:
Nit: line 46 we use `$PROJECT_NAME` and on line 47 we use `#PROJECT_NAME` 

(we can merge this and open a fresh PR as well, not a blocker!)